### DISCUSSION TITLE
Implement Manifest Request Codes

### DISCUSSION THREAD

**Discussion by nutterthanos** [ValvePython/steamctl#50] [General]
this needs to exist since when steam forces this steamctl won't be able to download any game unless the manifest for it is already on disk

**Comment by nutterthanos** [comment]
Ok I noticed it was implemented
Thanks

**Reply by rossengeorgiev** [reply]
Yes. It was implemented in `v1.3.0` as they forced it over the weekend, and potentially permanently.

**Comment by nutterthanos** [comment]
ah but they reverted it but at least its implemented if they do it again

**Comment by nutterthanos** [comment]
#54 it seems it only works for manifests currently listed in appinfo and depotdownloader has no problem at all.

**Comment by rossengeorgiev** [comment]
Manifest request codes should now work properly in `depot` commands. `steamctl>=1.4.0`

**Reply by nutterthanos** [reply]
um where is that release?

**Reply by rossengeorgiev** [reply]
Apparently in my imagination. Release uploaded to pypi `steamctl==0.9.5`
